terminus/generators/monolane_generator.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import print_function
+
+from collections import OrderedDict
+from math import atan2
+from math import degrees
+import yaml
+
+from .city_visitor import CityVisitor
+from models.intersection_node import IntersectionNode
+
+
+class MonolaneGenerator(CityVisitor):
+    def start_city(self, city):
+        """Called when a visitation of a new city starts."""
+        # Reset all of the class variables used until stop_city is called.
+        self.monolane = {
+            'maliput_monolane_builder': OrderedDict([
+                ('id', city.name),
+                ('lane_bounds', [-2, 2]),
+                ('driveable_bounds', [-4, 4]),
+                ('position_precision', 0.01),
+                ('orientation_precision', 0.5),
+                ('points', OrderedDict()),
+                ('connections', OrderedDict()),
+                ('groups', OrderedDict()),
+            ])
+        }
+        self.streets = []
+        self.point_name_by_street_node_dict = {}
+        self.intersection_index = 0
+        self.intersection_index_by_node = {}
+
+    def node_ident(self, street, node):
```

Comment:
It would be great to decouple the id->node mapping methods and instance variables (AFAIK only `point_name_by_street_node_dict`) to a separate object, like the `RNDFIdMapper`. We found a similar need for OpenDrive and created the (quite dummy) `OpenDriveIdMapper` so it would be great to have a third requirement for this to see if we can generalize it to a single class.